src/components/Tool/FixPhoneNumbers/FixPhoneNumbers.tsx

```diff
@@ -129,12 +130,12 @@ const FixPhoneNumbers: React.FC<Props> = ({
     [key: string]: PhoneNumberData;
   }>({});
 
-  const [sourceOptions, setSourceOptions] = useState<string[]>([appName]);
+  const [sourceOptions, setSourceOptions] = useState([appName]);
   const [showBulkConfirmModal, setShowBulkConfirmModal] = useState(false);
 
   // Create a mutable copy of the query data and store in the state
   useEffect(() => {
-    const existingSources = new Set<string>();
+    const existingSources = new Set<string | undefined>();
```

Comment:
Would it make more sense to leave this a Set of strings and just not add sources to the set if they are undefined?